pvlib/irradiance.py

```diff
@@ -994,6 +1006,103 @@ def king(surface_tilt, dhi, ghi, solar_zenith):
     return sky_diffuse
 
 
+def muneer(surface_tilt, surface_azimuth, dhi, ghi, dni_extra, b=5.73,
+           solar_zenith=None, solar_azimuth=None, projection_ratio=None):
+    '''
+    Determine sky diffuse irradiance on a tilted surface using the
+    Muneer model.
+
+    This Muneer transposition model is described in [1]_.
```

Comment:
```suggestion
    Determine the sky diffuse irradiance on a tilted surface using
    the Muneer (1990) model [1]_.
```
More succinct

Reply:
I had suggested this edit due to the warnings on index pages - these pages don't have the reference link and they show the first paragraph of the description.
https://github.com/pvlib/pvlib-python/pull/2184#discussion_r1732505092

Reply:
This suggested change should not be committed.